src/main/javadoc/doc-files/speclicense.html

```diff
@@ -12,8 +12,8 @@
       a) in the case of the initial Contributor, the initial content
          Distributed under this Agreement, and
 
-      b) in the case of each subsequent Contributor:
-         i) changes to the Program, and
+      b) in the case of each subsequent Contributor: 
```

Comment:
Should we disable the Jenkins job check for trailing spaces?  Or remove the trailing spaces?

Reply:
I'd vote just to remove them.